packages/integrations/mdx/README.md

```diff
@@ -103,6 +103,22 @@ const posts = await Astro.glob('./*.mdx');
 
 See [the official "how MDX works" guide](https://mdxjs.com/docs/using-mdx/#how-mdx-works) for more on MDX variables.
 
+### Exported properties
+
+Alongside your [MDX variable exports](#variables), we generate a few helpful exports as well. These are accessible when importing an MDX file via `import` statements or [`Astro.glob`](https://docs.astro.build/en/reference/api-reference/#astroglob).
+
+#### `file`
+
+The absolute path to the MDX file (e.g. `home/user/projects/.../file.md`).
+
+#### `url`
+
+The browser-ready URL for MDX files under `src/pages/` (e.g. `en/guides/markdown-content/`). For all other MDX files, this will be `undefined`.
+
+#### `getHeadings()`
+
+A function that returns all headings (i.e. `h1 -> h6` elements) of an MDX file. The response follows this type: `{ depth: number; slug: string; text: string }[]`. The `slug` corresponds to the generated ID for a given heading, and can be used for anchor links.
```

Comment:
What do you think of pulling the type out of the paragraph like this for easy reference?

I’ve also tweaked wording slightly: “the MDX file” because this returns headings for a specific file and adding “Each heading” to the second sentence to underline the fact you’re talking about an array entry.

```suggestion
**Returns:** `{ depth: number; slug: string; text: string }[]`

A function that returns an array of all headings (i.e. `h1 -> h6` elements) in the MDX file. Each heading’s `slug` corresponds to the generated ID for a given heading and can be used for anchor links.
```

Reply:
@delucis I actually expanded on [the markdown docs we have today](https://docs.astro.build/en/guides/markdown-content/#getheaders) 😁 Would love to update our MD docs to this draft! Replacing MDX with MD of course